src/background.js

```diff
@@ -113,7 +113,6 @@ async function handlePopupMessage(message) {
 			let result = {
 				disable: [
 					"*",
-					"edit*",
```

Comment:
This should stay here, because it ensures that the edit menu isn’t enabled when the content script can’t start (i.e. the page is privileged (`about:`, `chrome://` or `resource://`)).

Yes, I do realize that this is contained in the `*` selector above, but that one will be removed once the menu is fully implemented, while the `edit*` selector and any other selector dependant on the content script will be kept.